### PR TITLE
Italian - more text

### PR DESCRIPTION
**Pull request by ar-x**
via @andreafno 

Closes https://github.com/ar-x/turtlecoin.lol-translations/issues/7

### CODE REVIEW

**Comment by ar-x** [comment]
@andreafno this is just waiting for a review now.

**Comment by ar-x** [comment]
cc @4k4

File: italian/05_25_2018.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+https://turtlecoin.lol/it/brand/
+
+the navbar here
+
+7. Linee guida
+8. Asset
+9. Unisciti a noi
+10. Torna alla pagina principale
```
Comment (ar-x) [inline italian/05_25_2018.txt]: @andreafno is there a shorter version of line 10. This text would fold onto 2 lines (based on current font size) on mobile. Its ok if there isn't.

Comment (ar-x) [reply italian/05_25_2018.txt]: even 'Back' is fine I think.

Comment (andreafno) [reply italian/05_25_2018.txt]: @ar-x Ok done!

**Pull request merged**